extensions/default/src/DicomLocalDataSource/index.js

```diff
@@ -153,6 +155,7 @@ function createDicomLocalApi(dicomLocalConfig) {
                 StudyInstanceUID,
                 SeriesInstanceUID,
                 SOPInstanceUID,
+                frameIndex: isMultiframe ? index : undefined,
```

Comment:
Is there any reason to not include frameIndex if this is an image object?  An index of 1 is the correct index for images all the time, even for non-multiframes.

Reply:
done